docs/videos/healthcare-and-life-sciences/Factor-Xa-example/README.md

```diff
@@ -0,0 +1,43 @@
+# Factor Xa Example
+
+This example demonstrates a real life example of simulating the Factor Xa
+protein in water.
+
+It has been based on tutorials from Justin Lemhul (http://www.mdtutorials.com)
+and Alessandra Villa (https://tutorials.gromacs.org/).
+
+> **Note** This example has not been optimized for performance and is meant to
+> demonstrate feasibility of a real world example.
+
+## Instructions
+
+1. Deploy the HCLS blueprint
+
+   Full instructions are found [here](../README.md).
+
+1. SSH into the Slurm login node
+
+1. Create a submission directory
+
+   ```bash
+   mkdir factorxa_run01 && cd factorxa_run01
+   ```
+
+1. Copy the contents of this directory into the submission directory
+
+   ```bash
+   git clone https://github.com/GoogleCloudPlatform/hpc-toolkit.git
```

Comment:
Is git present by default in the login node?

Reply:
Yes. I have tested. I am not sure across all images, but at least the hpc+slurm image that is used in this blueprint.

PS. As a follow on I may come up with a way of pre-loading the example as part of the builder run. But I wanted to start simple before I try to get fancy.